.github/workflows/ci.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       - name: Lint
         run: pylint ./**/*.py
 
+      - name: Black Formatting
+        run: black . --check
+
```

Comment:
Could you match this to [MITxOnline's CI](https://github.com/mitodl/mitxonline/blob/main/.github/workflows/ci.yml#L68-L69). Doesn't matter much but in that case, the naming and definition would remain the same across our different apps.